src/components/DpDraggable/DpDraggable.stories.mdx

```diff
@@ -0,0 +1,28 @@
+import { Meta } from '@storybook/addon-docs'
+import DpDraggable from './DpDraggable'
+
+<Meta
+    title="Components/Draggable"
+    component={DpDraggable}
+/>
+
+# Draggable
+
+Use the Draggable component to change the order of items within a list.
+
+
+## Default usage
+
+```html
+<dp-draggable
+    v-model="currentList"
+    :opts="draggableOptions">
+    <div class="o-sortablelist__item u-pv-0_5 u-pl-0_5 border--top">
```

Comment:
Do we need the classes to illustrate the use case?

Reply:
https://github.com/demos-europe/demosplan-ui/pull/497/commits/81eb766a12bd1d601f0bedec40bcc9fe6b92c84e

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/497/commits/9fc6053707fe76cae13f9626097bc652e23bd500